src/sentry/utils/pytest/selenium.py

```diff
@@ -175,7 +196,13 @@ def percy(request):
 @pytest.fixture(scope='function')
 def browser(request, percy, live_server):
     driver_type = request.config.getoption('selenium_driver')
-    if driver_type == 'firefox':
+    if driver_type == 'chrome':
+        options = webdriver.ChromeOptions()
+        options.add_argument('headless')
+        options.add_argument('disable-gpu')
+        options.add_argument('window-size=1280x800')
+        driver = webdriver.Chrome(chrome_options=options)
+    elif driver_type == 'firefox':
```

Comment:
Do we run tests w/ firefox at all? Could also run firefox headless (maybe)

Reply:
nope we don't -- this was more if you needed to run locally, or for a hypothetical world where we did multi browser